e2e/features/networkManagement/NetworkManagement.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Network Management
+  Background: Add an account and navigate open Switch network modal
+    Given I go to page "wallet"
+    And I wait for "1 seconds"
+    And I click on a button with text "customNode"
+    Then "customNode" should be selected in "Switch network" dropdown
+    Given I click on text "Add network"
+    Then I should be redirected to route: 'wallet?modal=dialogAddNetwork'
+    And button with text "Add network" should be disabled
+    Given I type "custom_devnet" in "name"
+    And I type "http://devnet-service.liskdev.net:9901" in "serviceUrl"
+    And I wait for "1 seconds"
+    And I click on a button with text "Add network"
+    Then I should see 'Custom network added "custom_devnet"'
+
+  Scenario: Edit network
```

Comment:
```suggestion
  Scenario: Edit custom network
```

Reply:
@eniolam1000752 If only custom networks can be edited, is it necessary to rename this is as "edit custom network"?

Reply:
Agree changed to more Correct, Edit network name